src/components/twilio-voice-dialer/routes.js

```diff
@@ -55,9 +60,29 @@ router.post('/twiml', (req, res) => {
     .send(twiml.toString());
 });
 
-router.post('/conference-events', (req, res) => {
-  // Handle conference events
-  console.log(req.body);
+router.post('/conference-events', async (req, res) => {
+  const { ConferenceSid, StatusCallbackEvent } = req.body;
+
+  if (StatusCallbackEvent === 'participant-join') {
+    const participants = await client.conferences(ConferenceSid).participants.list();
+
+    // Send the conference sid and the other participants' callSids to any client leg.
+    // The sids will be used to update the participant resource such as putting it on hold.
+    participants.forEach((currentParticipant) => {
+      if (currentParticipant.label.split('-')[0] === 'client') {
+        participants.filter((p) => p.callSid !== currentParticipant.callSid).forEach(async (otherParticipant) => {
+          await client
+            .calls(currentParticipant.callSid)
+            .userDefinedMessages
+            .create({content: JSON.stringify({
+              conferenceSid: ConferenceSid,
+              callSid: otherParticipant.callSid,
+            })});
+        });
+      }
```

Comment:
Just calling this out @mhuynh5757 @kpchoy. This can be optimized such as running all async calls in parallel, then return the response. However, we should probably not micro-optimize at this stage and sacrifice readability.